assembler/manual.html

```diff
@@ -214,8 +214,8 @@ <h3>2.2 Downloading SPAdes binaries for Mac</h3>
         <li><code>hammer</code>  (read error correcting module for Illumina reads)</li>
         <li><code>ionhammer</code>  (read error correcting module for IonTorrent reads)</li>
         <li><code>spades</code>  (assembly module)</li>
-        <li><code>bwa-spades</code>  (<a href="http://bio-bwa.sourceforge.net" target="_blank">BWA</a> alignment module which is required for mismatch correction)</li>
-        <li><code>corrector</code>  (mismatch correction module)</li>
+        <li><code>bwa-spades</code>  (<a href="http://bio-bwa.sourceforge.net" target="_blank">BWA 0.7.12-r1039</a> bundled binary which is required for mismatch correction of final contigs)</li>
```

Comment:
ditto

Reply:
You are welcome to drop the two iinstances of this section out of the total three existing in the manual. Emphasizing the bwa version is important, I would even ask you whether this is original or modified version. Why cannot spades use a system-wide bwa?

Reply:
Because then the results may be different depending on the version of system BWA.

Reply:
Then it is even more important to say what version it depends on.